### PR TITLE
Discount calculator

### LINKED ISSUE #59: discount calculator
Adding a discount calculator page to calculate the saving and the discounted price.

### PR DESCRIPTION
**Pull request by anuamloh07**
Closes #59

### CODE REVIEW

**Comment by anuamloh07** [comment]
Discount calculator feautre whihc can be used in various shopping malls, etc and in daily life as well to get a better idea of the savings

File: index.html
Comment (RuanAragao) [inline index.html:1]: Correct the path. Use slash instead of backslash. ("./calculator/my-calculator/")

**Review by RuanAragao — CHANGES_REQUESTED** [review]
**Fix:**
The _Kebab case_ is required for all files and directories. 
Ex.: `./my-directory/my-file.ext`

**Code quality tip:**
All functions, variables and classes always have descriptive, not generic, names. This way it is clearer what the function does.

Exemple:
```javascript
// Bad name
function f1(a, b) {
  const total = a + b;
  return total;
}

// Good name
function sum(a, b) {
  const total = a + b;
  return total;
}
```

File: index.html [hunk 1]
```diff
@@ -31,6 +31,7 @@ <h1>Calculators</h1>
   <a href="./calculators/tip-calculator">Tip Calculator</a><br />
   <a href="./calculators/PX-to-EM-converter/index.html">Pixel to Em Converter </a><br />
   <a href="./calculators/count-by-weight/index.html">Count by Weight</a><br />
+  <a href="./calculators/discount-calculator/index.html">Discount Calculator</a>
```
Comment (RuanAragao) [inline index.html]: @anuamloh07 need a `<br />` end of line

Comment (RuanAragao) [reply index.html]: I added the accepted tag in the PR, but I need you to correct this to do the merge. Thanks!

**Review by RuanAragao — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (RuanAragao) [reply index.html]: I fix this

**Pull request merged**